test/clj/puppetlabs/trapperkeeper/services/webserver/jetty9_config_test.clj

```diff
@@ -93,8 +103,37 @@
 
     (is (expected-https-config?
           (merge valid-ssl-pem-config
-                 {:ssl-host "foo.local" :ssl-port 8001 :request-header-max-size 16192})
-          {:https {:host "foo.local" :port 8001 :request-header-max-size 16192}})))
+                 {:ssl-host "foo.local"
+                  :ssl-port 8001
+                  :request-header-max-size 16192})
+          {:https {:host "foo.local"
+                   :port 8001
+                   :request-header-max-size 16192}}))
+
+    (is (expected-https-config?
+          (merge valid-ssl-pem-config
+                 {:ssl-host "foo.local"
+                  :ssl-port 8001
+                  :so-linger-seconds 22})
```

Comment:
I'd agree that it would be better to expose this as milliseconds, however, the underlying API only supports seconds for granularity.  The value that you pass in to the Jetty ServerConnector.setSoLingerTime() - not documented in the method signature - is divided by 1000 as it is passed to the underlying Java Socket:

https://github.com/eclipse/jetty.project/blob/jetty-9.1.0.v20131115/jetty-server/src/main/java/org/eclipse/jetty/server/ServerConnector.java#L355

JavaDocs for the Socket.setSoLinger method include:

```
* Enable/disable SO_LINGER with the specified linger time in seconds.
```

It makes sense that Java constrains this to seconds in that the underlying BSD SO_LINGER option is also expressed in seconds.  From the SO_LINGER documentation in http://man7.org/linux/man-pages/man7/socket.7.html:

```c
struct linger {
  int l_onoff;    /* linger active */
  int l_linger;   /* how many seconds to linger for */
};
```